org.eclipse.acute/plugin.xml

```diff
@@ -265,4 +265,18 @@
             type="java.lang.Object">
       </propertyTester>
    </extension>
+   <extension
+         point="org.eclipse.core.runtime.preferences">
+      <initializer
+            class="org.eclipse.acute.AcutePreferenceInitializer">
+      </initializer>
+   </extension>
+   <extension
+         point="org.eclipse.ui.preferencePages">
+      <page
+            class="org.eclipse.acute.AcutePreferencePage"
+            id="org.eclipse.acute.preferencePage"
+            name="aCute">
```

Comment:
aCute is an internal project name. I imagine end users would like to see ".NET Core" or something like that as preference page title.